test/elastic_box/plotting/Ncoll_plotting.py

```diff
@@ -41,14 +39,21 @@
             'T': "temperature $T$, GeV",
             'dt': "time step size $dt$, fm/c",
             'Kn': "Knudsen number"}
-y_label =  ('$\\frac{N_{coll}^{SMASH}}{N_{coll}^{theory}} \, = \,$'
-            '$\\frac{1}{N \, N_{test} \,/ 2} \cdot '
-            '\\frac{N_{coll}}{N_{ev} t_{tot}} \cdot '
-            '\\frac{V}{\sigma N \, \langle v \\rangle}$')
 
-def plot_data(input_txt_file):
+y_label =  ('${N_{coll}^{SMASH}}\\left/{N_{coll}^{theory}} \\right.$')
+y_label_defintion =  ('$\\frac{N_{coll}^{SMASH}}{N_{coll}^{theory}} \, = \,$'
+                      '$\\frac{1}{N \, N_{test} \,/ 2} \,'
+                       '\\frac{N_{coll}}{N_{ev} t_{tot}}\,'
+                       '\\frac{V}{\sigma N \, \langle v \\rangle}$')
+
+
+def plot_data(input_txt_file, plot_position, plot_color):
+
+    plt.subplot(plot_position) #sublot(AnzahlZeilen AnzahlSpalten BildNummer)
```

Comment:
What about English?

Reply:
Removed comment and squashed into previous commit.